Experimental/src/org/sleuthkit/autopsy/experimental/autoingest/FileExportRuleSet.java

```diff
@@ -337,12 +351,12 @@ boolean conditionsAreEqual(Rule that) {
                 return false;
             }
             this.fileSizeConditions.sort(null);
-            that.fileSizeConditions.sort(null);
+            that.sortFileSizeConditions();
```

Comment:
It might be a tiny bit more readable to call this.sortFileSizeConditions as well. Ditto on line 359.

Reply:
good call